common/src/main/java/org/dash/wallet/common/util/GenericUtils.kt

```diff
@@ -129,15 +129,14 @@ object GenericUtils {
      * @param fiatValue
      * @return
      */
-    fun formatFiatWithoutComma(fiatValue: String): String? {
-        val fiatValueContainsCommaWithDecimal = fiatValue.contains(",") && fiatValue.contains(".")
+    fun formatFiatWithoutComma(fiatValue: String): String {
+        val fiatValueContainsCommaWithDecimal = fiatValue.contains(".") && fiatValue.contains(",")
+
         return if (fiatValueContainsCommaWithDecimal) {
-            fiatValue.replace(
-                ",".toRegex(),
-                ""
-            )
+            fiatValue.replace(",", "")
         } else {
-            fiatValue.replace(",".toRegex(), ".")
+            fiatValue.replace(",", ".")
+                .replace("٫", ".")
```

Comment:
At first, I wanted to parse the input in a "proper" manner with the current device locale, however, this proved problematic because `formatFiatWithoutComma` often receives input in a US formatted manner.

At some point, we might want to unify the way decimal separators are presented to the users in different locales, but it will require a bit of work and extensive testing. For now, this fix should suffice.